lib/extensions.js

```diff
@@ -359,6 +370,23 @@ function getVersionInfo(binding) {
   ].join(eol);
 }
 
+/**
+ * Gets the platform variant, either an empty string for non-Linux systems or the
+ * libc implementation (glibc|musl) on Linux.
+ *
+ * @api public
+ */
+
+function getPlatformVariant() {
+  if (process.platform != 'linux') {
+    return '';
+  }
+  if (fs.readFileSync(process.execPath).indexOf('libc.musl-x86_64.so.1') != -1) {
+    return 'musl';
+  }
+  return 'glibc';
```

Comment:
Probably better to just return empty unless it's musl. Would simplify the code above